miniwallet/test/Mini1155.ts

```diff
@@ -0,0 +1,489 @@
+      // expect(await this.mini1155.royaltyInfo(1, 1)).to.equal(this.deployer.address, 0)
+      expect(await this.mini1155.saleIsActive()).to.equal(deploymentConfig.saleIsActive)
+      expect(await this.mini1155.saleStarted()).to.equal(false)
+      expect(await this.mini1155.salt()).to.equal(deploymentConfig.salt)
+      expect(await this.mini1155.standardTokenId()).to.equal(deploymentConfig.s.tokenId)
+      expect(await this.mini1155.supportsInterface(Constants.InterfaceId.ERC2981_NFT_ROYALTY_STANDARD)).to.equal(true)
+      expect(await this.mini1155.symbol()).to.equal(deploymentConfig.symbol)
+      expect(await this.mini1155.totalSupply(deploymentConfig.s.tokenId)).to.equal(0)
+      expect(await this.mini1155.uri(deploymentConfig.s.tokenId)).to.equal('ipfs://QmPcY4yVQu4J2z3ztHWziWkoUEugpzdfftbGH8xD49DvRx/1.json')
+    })
+
+    it('Mini1155-2 Configuration Validation', async function () {
+      // Set Standard and Rare Tokens
+      // await mini1155configure({ mini1155: this.mini1155, revenueAccount: await this.mini1155.owner() })
+      await mini1155configure({ mini1155: this.mini1155, collectionConfig: c1 })
+      // Check all readOnlyFunctions
+      expect(this.mini1155.address).to.equal(
+        '0x5FbDB2315678afecb367f032d93F642f64180aa3'
+      )
+      // Test Token Configuration
+      expect(await this.mini1155.owner()).to.equal(this.deployer.address)
+      expect(await this.mini1155.metadataFrozen()).to.equal(false)
+      expect(await this.mini1155.standardTokenId()).to.equal(c1.s.tokenId)
+      expect(await this.mini1155.maxSupply(c1.s.tokenId)).to.equal(c1.s.maxSupply)
+      expect(await this.mini1155.maxPersonalCap(c1.s.tokenId)).to.equal(c1.s.personalCap)
+      expect(await this.mini1155.uri(c1.s.tokenId)).to.equal('ipfs://QmPcY4yVQu4J2z3ztHWziWkoUEugpzdfftbGH8xD49DvRx/1.json')
+      expect(await this.mini1155.rareTokenId()).to.equal(c1.r.tokenId)
+      expect(await this.mini1155.maxSupply(c1.r.tokenId)).to.equal(c1.r.maxSupply)
+      expect(await this.mini1155.maxPersonalCap(c1.r.tokenId)).to.equal(c1.r.personalCap)
+      expect(await this.mini1155.uri(2)).to.equal('ipfs://QmPcY4yVQu4J2z3ztHWziWkoUEugpzdfftbGH8xD49DvRx/2.json')
+      // Although configured tokens don't exist till minted
+      expect(await this.mini1155.exists(c1.s.tokenId)).to.equal(false)
+      expect(await this.mini1155.exists(c1.r.tokenId)).to.equal(false)
+      // Test View Functions
+      const owner = await this.mini1155.owner()
+      // const tid = 1
+      expect(await this.mini1155.balanceOf(owner, c1.s.tokenId)).to.equal(0)
+      expect((await this.mini1155.balanceOfBatch([owner, owner], [1, 2])).map(x => x.toString())).to.deep.equal(['0', '0'])
+      expect(await this.mini1155.baseUri()).to.equal('ipfs://QmPcY4yVQu4J2z3ztHWziWkoUEugpzdfftbGH8xD49DvRx/')
+      expect(await this.mini1155.contractURI()).to.equal('ipfs://QmdKB6d1zT7R8dNmEQzc6N1m5p2LDJZ66Hzu8F4fGhdVrq')
```

Comment:
these specific project urls all need to be sanitized and removed from tests and default config parameters